portal-bridge/src/cli.rs

```diff
@@ -92,6 +93,13 @@ pub struct BridgeConfig {
         help = "Data provider for consensus layer data. (\"pandaops\" / local node url)"
     )]
     pub cl_provider: Provider,
+
+    #[arg(
+        long = "el-provider-request-limit",
```

Comment:
Why not just make it a "provider-daily-request-limit" and apply it to both el & cl providers?

Reply:
I guess beacon only uses CL and history only uses EL so this wouldn't be an issue, but it could become an issue when we support state. I guess we can just do it naively now and fix it later.

I would prefer specifing them seperately through for that reason.

Reply:
I am going to make the change suggested with the assumption bridges only use CL or EL, but not both. If they use both I would want to have the parameters seperate, but we can do that in a future PR if needed.

Reply:
I'm not sure that's a safe assumption, for example our `"latest"` bridges run both `history` & `state` network bridges, meaning they send requests to both the el & cl provider

Reply:
Ok I made it so you can specify a limit for el and a limit for cl pr should be ready for another review.